devkit/sandbox/package.json

```diff
@@ -1,15 +1,27 @@
 {
-  "name": "@rpch/sandbox",
-  "version": "0.1.1",
-  "private": true,
-  "license": "LGPL-3.0",
-  "scripts": {
-    "build": "exit 0",
-    "dev": "exit 0",
-    "format": "exit 0",
-    "lint": "exit 0",
-    "test": "exit 0",
-    "start": "./src/start.sh",
-    "stop": "./src/stop.sh"
-  }
-}
\ No newline at end of file
+    "name": "@rpch/sandbox",
+    "version": "0.2.0",
```

Comment:
this would be updated later by changeset tool

Reply:
Ok